### ISSUE TITLE
Make progress monitoring screen of complainer

### ISSUE THREAD

**Issue by Jiggy9** [Jiggy9/JCT#25]
Once approved sms sent to perticular complaint by staff and progress monitoring start, make this screen

**Comment by EfrataAron** [comment]
for the sms sending, setmore allows sending sms but it requires payment. you can try setmore for that

**Comment by Jiggy9** [comment]
> for the sms sending, setmore allows sending sms but it requires payment. you can try setmore for that

No, it's free if use firebase as there's already firebase service if it, it just need to be implemented
